diff --git a/src/core/lib/utils/get-call-stack.cpp b/src/core/lib/utils/get-call-stack.cpp
--- a/src/core/lib/utils/get-call-stack.cpp
+++ b/src/core/lib/utils/get-call-stack.cpp
@@ -30,5 +30,5 @@
 //==================================================================================
 #include "utils/get-call-stack.h"
 
-#if defined(__linux__) && defined(__GNUC__)
+#if defined(__linux__) && defined(__GNUC__) && !defined(__ANDROID__)
 // clang-format off
 #include "utils/demangle.h"
